static/mainapp/css/layout.css

```diff
@@ -16,6 +16,9 @@ body {
   flex-direction: column !important;
   align-items: center !important;
 }
+.navbrandimg{
+  width: 100px;
```

Comment:
The width of approx 130px suits better for this, the logo is appearing too small in large screen mode.